Editor/SnapZone/SnapZoneSettings.cs

```diff
@@ -31,24 +31,39 @@ public class SnapZoneSettings : ScriptableObject
         private Material highlightMaterial;
 
         /// <summary>
-        /// This color is used when an <see cref="InteractableObject"/> is hovering a <see cref="SnapZone"/>.
+        /// This color is used when a valid <see cref="InteractableObject"/> is hovering a <see cref="SnapZone"/>.
```

Comment:
ValidationColor probably should be ValidColor since it is also InvalidColor.
Validation is the process and therefore could also be invalid in the end

Reply:
should probably be that, but this is an api break :|